assets/js/components/Main/index.js

```diff
@@ -33,9 +41,28 @@ class MainPage extends Component {
 
     render() {
         const {activeTab} = this.state
+        const {intl, tunnistamoUser, user} = this.props
+        const hasImage = Boolean(user.image)
 
         return (
             <div className="oma-main">
+                <div className="greetings-container">
+                    {
+                        hasImage ? (
+                            <div className="greetings-user-image" >
+                                <img src={user.image} alt="profile" />
+                            </div>
+                        ) : (
+                            <div className="greetings-icon-container">
+                                <HelIcon iconName="user-o"></HelIcon>
+                            </div>
+                        )
+                    }
+
+                    <div className="greetings-text-container">
+                        <h2 className="greetings-text"><span>{intl.formatMessage({id: 'app.hello'})} {tunnistamoUser.first_name}</span></h2>
```

Comment:
I think that the username is a part of the message. I would rename the string to something like `app.userGreeting` and then the message would be 'Hello {user}'